### ISSUE TITLE
Camera stream only working from Device Tiles view in browser

### ISSUE THREAD

**Issue by jordansmith74** [tonesto7/nest-manager#653]
I am unable to stream live video from Nest cameras from within the Smartthings classic app, but if I go to NST Manager > View Device Tiles > I am able to see all 4 cameras working and live. 

Error message: "Camera cannot be found. We are trying to connect."
No errors in logs

- [x ] You followed the Nest Developer documentation here? [Nest Developer Account Creation](https://cdn.rawgit.com/tonesto7/nest-manager/master/Documents/html/Create_Dev_Account.html)
- [x ] That _OAuth_ is Enabled for the SmartApp under the IDE.
- [x ] The SmartApps and *_ALL_* (5) Devices Handlers are using the latest code available.
- [x ] You have reviewed the [NST Manager Wiki](http://thingsthataresmart.wiki/index.php?title=NST_Manager) for potential fixes.
- [x ] That ALL SmartApps and Device Handlers have been _Published for You_ in the IDE.

## About Your Setup:
* How many Thermostats?: 0
* How many Protects?: 0
* How many Cameras?: 4
* Using the Nest Presence Device?: No
* Using the Nest Weather Device?: No
* iOS or Android?: iOS
* Mobile App Version: Classic 2.17.1 (1442)

## Expected Behavior
Expecting to see the video player of live camera footage

## Current Behavior
Error: Camera cannot be found. We are trying to connect.

## Context
I did a complete re-install and everything is working other than when I click the play button to view live video.

IMPORTANT === When I go to NST Manager app in Smartthings and click on View Device Tiles, I am able to click the play button and see the live video. Everything works perfectly from that browser screen but from within the Smartthings classic app it is not pulling up. Since the browser view of the tiles is working I feel like I have set everything up correctly. I looked everywhere for related issues but this seemed pretty unique.

link: https://graph-na04-useast2.api.smartthings.com/api/smartapps/installations/........

**Comment by mvevitsis** [comment]
Same here. Did you ever fix it?

**Comment by jordansmith74** [comment]
No, I never did. I didn't know where else to look since the stream is working when you go directly to the link. Some iOS security thing maybe. Let me know if you ever find a solution.

Thanks
